src/components/AddTaskUI.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChangeEvent } from 'react';
+
+interface AddTaskUIProps {
+  isBlank: boolean;
+  handleTaskInput: (e: ChangeEvent<HTMLInputElement>) => void;
+  newTask: string;
+}
+
+function AddTaskUI({ isBlank, handleTaskInput, newTask }: AddTaskUIProps) {
+  return (
+    <>
+      <div className="relative w-full">
+        <input
+          className={`w-full border rounded-md p-2 
+          ${isBlank ? 'border-red-600' : 'border-black'}`}
+          placeholder="New task.."
+          type="text"
+          name="newTask"
+          id="newTask"
+          onChange={handleTaskInput}
+          value={newTask}
+        />
+        {isBlank && (
```

Comment:
Nitpick 🪡:  I prefer to do ternaries versus `&&`. Sometimes people will do `someArray.length && <Component />` and you'll see the number 0 appear in the UI lol.

Reply:
updated in: 

`AddTaskUI.tsx`
`DeleteModalUI.tsx`